demos/speech_recognition_offline_demo/python/speech_recognition_offline_demo.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# Workaround to import librosa on Linux without installed libsndfile.so
+try:
+    import librosa
+except OSError:
+    import sys
+    import types
+    sys.modules['soundfile'] = types.ModuleType('fake_soundfile')
+    import librosa
+    del sys.modules['soundfile']
```

Comment:
I don't think you're accomplishing anything by deleting it. If librosa imports it, then it will retain a reference to it, so the module object will stick around anyway. You might as well keep the program state consistent by retaining it in `sys.modules`.

Reply:
Removed